db/academyPortal_2020-05-20.sql

```diff
@@ -0,0 +1,345 @@
+# ************************************************************
```

Comment:
i cannot tell what has changed because you have deleted the old one and added a new one, please overwrite the old one so i can see the changes